OP-340 Updating Ward fix

dnlebgwa opened the pull request:
Changing Ward hibernate core code to return saved object instead of boolean to fix OP-340

dbmalkovsky commented on src/test/java/org/isf/ward/test/Tests.java:
```diff
@@ -148,11 +172,10 @@ public void testIoUpdateWard() {
 			Ward updateWard = wardIoOperationRepository.findOne(code);
 
 			// then:
-			assertTrue(result);
```
FWIW, by the fact that `assertTrue()` and `catch-try-fail` are present it appears that the `develop` branch this was based on is significantly out of date.

dnlebgwa replied:
@dbmalkovsky  sorry about that. I have updated my branch and re-merged the changes. Is this ok?

dbmalkovsky replied:
There are still issues; see my comments in the source files.

dbmalkovsky commented on src/main/java/org/isf/ward/service/WardIoOperations.java:
```diff
@@ -11,13 +11,36 @@
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
+<<<<<<< HEAD
```
This conflict needs to be resolved.

dbmalkovsky commented on src/main/java/org/isf/ward/service/WardIoOperations.java:
```diff
@@ -11,13 +11,36 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
+=======
```
This conflict needs to be resolved.

dbmalkovsky commented on src/main/java/org/isf/ward/manager/WardBrowserManager.java:
```diff
@@ -142,10 +142,14 @@ protected void validateWard(Ward ward, boolean insert) throws OHServiceException
 	 * In case of error a message error is shown and a <code>false</code> value is returned.
 	 * @param ward the ward to store.
-	 * @return <code>true</code> if the ward has been stored, <code>false</code> otherwise.
+<<<<<<< HEAD
```
This is still a conflict; needs a resolution.  There are other occurrences too.

dnlebgwa commented:
Thanks @dbmalkovsky. I believe I have addressed your comments in my latest commit
https://github.com/informatici/openhospital-core/pull/253/commits/2547a4dfb06712cef71208445578716c96f45da0
